### PR TITLE
Fixes docs to include single wrappers for full-width text inputs

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Docs: Nests full width modifier inputs in their own m-form-field divs.

### CODE REVIEW

File: src/cf-forms/usage.md [hunk 1]
```diff
@@ -331,7 +333,9 @@ Generally this is only useful for documentation purposes.
            type="text"
            id="full-textinput-example"
            value="Lorem ipsum">
-    <br><br>
+</div>
+<br>
```
Comment (Scotchester) [inline src/cf-forms/usage.md]: For the code example, should we omit the `<br>` and just have a blank line here?

Comment (jimmynotjim) [reply src/cf-forms/usage.md]: It's from way back before these were ever wrapped. I'd say yes

Comment (anselmbradford) [reply src/cf-forms/usage.md]: Does not work:
screen shot 2017-10-03 at 6 00 44 pm

**Comment by Scotchester** [comment]
Maybe I commented in the wrong place, but I'll taking about the sample
code, not the coffee that renders the live example.

**Comment by anselmbradford** [comment]
> Maybe I commented in the wrong place, but I'll taking about the sample
code, not the coffee that renders the live example.

Ooo. Do we want to go down the road of where the code example and rendered code are different? I'm not sooo keen on that. I'd think it shouldn't be confusing of why copying the code example does not look exactly like the live example. If we don't want the `<br>` what do you think about providing a code example under each instead of one code example combined together?

**Comment by Scotchester** [comment]
I think realistically, people won't be copying both fields together, they'll select one or the other and copy that. But if we want to be 0% error-prone, I agree with the idea of doing the code example under each.

**Comment by anselmbradford** [comment]
> I think realistically, people won't be copying both fields together, they'll select one or the other and copy that. But if we want to be 0% error-prone, I agree with the idea of doing the code example under each.

Done!

**Comment by anselmbradford** [comment]
screen shot 2017-10-04 at 11 02 00 am

**Comment by Scotchester** [comment]
:shipit:

**Pull request merged**